### PR TITLE
Issue #13195: Mobile page rendering requires extra font difference for properties

### LINKED ISSUE #13195: Mobile page rendering requires extra font difference for properties
Detected at https://github.com/checkstyle/checkstyle/pull/13103#issuecomment-1585814118

There only pop point to improve is to make name of property to be italic to look different than description.

Screenshot_20230610-135311

### PR DESCRIPTION
**Pull request by stoyanK7**
Resolves #13195

### CODE REVIEW

**Comment by stoyanK7** [comment]
Github, generate site

**Review by nrmancuso — APPROVED** [review]
I do not have experience with CSS/ web dev in general, but change looks good to me.

**Comment by nrmancuso** [comment]
@rnveach please take a look, I know you are more experienced with web dev.

**Comment by romani** [comment]
Github, generate site

**Comment by romani** [comment]
Previous website is not opening for me

**Comment by stoyanK7** [comment]
@romani Weird. Works for me. Here is a direct link to some properties: https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/0b86150_2023191601/checks/annotation/annotationlocation.html#AnnotationLocation_Properties

**Comment by romani** [comment]
It works now

File: src/site/resources/css/site.css [hunk 1]
```diff
@@ -163,4 +163,8 @@ a[title="toTop"] {
     display: block;
     text-align: left;
   }
+
+  section[id$="_Properties"] .wrapper .bodyTable tr td:first-child {
```
Comment (rnveach) [inline src/site/resources/css/site.css:167]: Looks like there is some issue in new pages if the ID starts with `_`. Originally IDs were like `id="AnnotationLocation_Properties"` before we split. The `_` was added to provide a split between check name and section name. The `_` was not removed when it should have been.

**Review by rnveach — CHANGES_REQUESTED** [review]

File: src/site/resources/css/site.css (hunk 1 above)
Comment (romani) [reply src/site/resources/css/site.css:167]: Yes, better to be without leading `_`

Comment (stoyanK7) [reply src/site/resources/css/site.css:167]: Done.

**Comment by stoyanK7** [comment]
Github, generate site

File: src/site/resources/css/site.css (hunk 1 above)
Comment (rnveach) [reply src/site/resources/css/site.css]: @stoyanK7 I don't see any other code changes but removing the `_`. Is this a `find` and not a complete `match`?

Looking at https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/b00c60f_2023120428/checks/annotation/annotationlocation.html#AnnotationLocation_Description
It seems we still have the same pattern `check name _ section name`. I don't see the point in duplicating the check name with the same page name.
These `href`s should be changed for the new page style.

Comment (rnveach) [reply src/site/resources/css/site.css]: This should be a complete match when the `href`s are fixed.

Comment (stoyanK7) [reply src/site/resources/css/site.css]: Mentioned in issue.

**Comment by romani** [comment]
@rnveach , it looks like ready for final review.

**Review by rnveach — APPROVED** [review]

**Pull request merged**